sdk/servicebus/service-bus/changelog.md

```diff
@@ -1,3 +1,10 @@
+# 2019-04-24 1.0.0-preview.3
+
+- README updated with API reference links
```

Comment:
Not worth calling out IMO.

Reply:
especially not as a first point :) will move it to the bottom
the main point I wanted to get across that we now have API reference docs!